api/src/display.rs

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+//! Character encodings for public addresses and request types for easy
+//! cut-and-paste.
+
+use crate::printable;
+use crc::crc32;
+use failure::Fail;
+use protobuf::{parse_from_bytes, Message};
+
+/// Decoding / encoding errors
+#[derive(Clone, Debug, Eq, PartialEq, Fail)]
+pub enum Error {
+    /// Protocol buffer could not be serialized
+    #[fail(display = "Could not serialize protocol buffer: {:?}", _0)]
+    SerializationFailure(String),
```

Comment:
In other places we omitted suffixes such as failure or error since this is implied.